parallel_wavegan/models/style_melgan.py

```diff
@@ -219,7 +218,7 @@ def inference(self, c, normalize_before=False):
         noise_size = (
             1,
             self.in_channels,
-            math.ceil(c.size(2) / self.noise_upsample_factor),
+            (c.size(2)-1) // self.noise_upsample_factor + 1,
```

Comment:
Note that both of `c.size(2)` and `self.noise_upsample_factor` are intergers. So that we can replace `math.ceil()` operation like the following code (We may use `torch.ceil()` function instead of `math.ceil()`. But it takes and returns tensor, not a scalar value.)

```
>>> import math
>>> noise_upsample_factor = 5
>>> for c_size in range(15):
...     before = math.ceil(c_size / noise_upsample_factor)
...     after = (c_size-1) // noise_upsample_factor + 1
...     print(f"before: {before}, after: {after}")
...
before: 0, after: 0
before: 1, after: 1
before: 1, after: 1
before: 1, after: 1
before: 1, after: 1
before: 1, after: 1
before: 2, after: 2
before: 2, after: 2
before: 2, after: 2
before: 2, after: 2
before: 2, after: 2
before: 3, after: 3
before: 3, after: 3
before: 3, after: 3
before: 3, after: 3
```